web/src/backend/jwts.ts

```diff
@@ -35,6 +35,28 @@ if (!GENERAL_SECRET_KEY) {
 
 // ANCHOR: -----------------HASURA JWTs--------------------------
 
+/**
+ * Generates a 1-min JWT for the `service` role (only for internal use from Next.js API)
+ * @returns
+ */
+export const generateReviewerJWT = async (): Promise<string> => {
```

Comment:
given we're still not using this, best to remove

Reply:
there's actually a misunderstanding here, the reviewer role calls the custom action, the custom action in Hasura calls the backend, the backend always interact with Hasura (for now at least) with the `service` role

Reply:
Sent you a message on slack but we are using a reviewer header in the client we use from verify app so we can enforce the reviewer permissions we set previously. I think this is better since we can add more granular permissions based on usecase